ambry-clustermap/src/main/java/com.github.ambry.clustermap/Partition.java

```diff
@@ -27,18 +27,18 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static com.github.ambry.clustermap.ClusterMapUtils.*;
+
 
 /**
  * A Partition is the unit of data management in Ambry. Each Partition is uniquely identifiable by an ID. Partitions
  * consist of one or more {@link Replica}s. Replicas ensure that a Partition is available and reliable.
  */
-public class Partition extends PartitionId {
+class Partition extends PartitionId {
```

Comment:
Yes. Added docs for `Partition`, `Replica`, `Disk`,`DataNode` and `DataCenter` - all static cluster manager related classes.